### DISCUSSION TITLE
Communicate with GLPI

### DISCUSSION THREAD

**Discussion by lyonelf** [amidaware/tacticalrmm#1548] [Ideas]
Hello,

Is there in the roadmap a plugin creation to have API share informations btw tactical and GLPI?

**Comment by dinger1986** [comment]
no, theres been afew hinting at having something in place on discord but nothing yet, happy to discuss with glpi if there are?

https://github.com/jpros/GLPI-TacticalRmmPlugin Allows you to open the machine via tactical from glpi
